src/main/java/datawave/query/util/MetadataHelper.java

```diff
@@ -1215,6 +1289,38 @@ public Long getCountsByFieldInDayWithTypes(String fieldName, String date, final
     protected HashMap<String,Long> getCountsByFieldInDayWithTypes(Entry<String,String> identifier) throws TableNotFoundException, IOException {
         String fieldName = identifier.getKey();
         String date = identifier.getValue();
+        boolean foundCompressedValue = false;
+        
+        Scanner scanner = ScannerHelper.createScanner(connector, metadataTableName, auths);
+        scanner.fetchColumnFamily(ColumnFamilyConstants.COLF_F);
+        scanner.setRange(Range.exact(fieldName));
+        
+        IteratorSetting cqRegex = new IteratorSetting(50, RegExFilter.class);
+        RegExFilter.setRegexs(cqRegex, null, null, MetadataHelper.COL_QUAL_PREFIX + ".*", null, false);
+        scanner.addScanIterator(cqRegex);
+        
+        final HashMap<String,Long> datatypeToCounts = Maps.newHashMap();
+        for (Entry<Key,Value> countEntry : scanner) {
+            foundCompressedValue = true;
+            DateFrequencyValue dateFrequencyValue = new DateFrequencyValue();
+            TreeMap<YearMonthDay,Frequency> dateFrequencies = dateFrequencyValue.deserialize(countEntry.getValue());
+            Long sum;
+            if (dateFrequencies.size() > 0 && dateFrequencies.get(new YearMonthDay(date)) != null) {
```

Comment:
I got this exception here: NumberFormatException: For input string: "comp".  It appears you are trying to handle the old format here but we should be processing the aggregated keys.

Reply:
oh wait, it is the date field that is "comp..." which is passed into this method.  Something else is wrong.....

Reply:
Its a test case that is bogus.  Please ignore this one.  Sorry.